sas-format/python-formats/sas/format.py

```diff
@@ -1,10 +1,7 @@
 from dataiku.customformat import Formatter, OutputFormatter, FormatExtractor
 from dataiku.base.java_link import LinkedInputStream
 from collections import OrderedDict
-
 import pandas as pd
-import json, base64
-import types
```

Comment:
Not sure if you're already familiar with this, but one of the nice things about Pycharm is you can do: Code | Inspect Code and quickly find unused imports, I try to do it once before a major checkin